sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/TestPubsubSignal.java

```diff
@@ -277,7 +303,7 @@ public void waitForSuccess(Duration duration) throws IOException {
     SubscriptionPath resultSubscriptionPath =
         PubsubClient.subscriptionPathFromName(
             pipelineOptions.getProject(),
-            "result-subscription-" + String.valueOf(ThreadLocalRandom.current().nextLong()));
+            "result-subscription-" + ThreadLocalRandom.current().nextLong());
```

Comment:
And this one?

Reply:
This one actually not required, just fixed what Idea highlighted me

Reply:
Reverted